tests/store/_unity_catalog/model_registry/test_utils.py

```diff
@@ -0,0 +1,75 @@
+import json
+import pytest
+from unittest import mock
+from unittest.mock import ANY
+
+from mlflow.entities.model_registry import ModelVersion, RegisteredModel, RegisteredModelAlias
+from mlflow.protos.databricks_uc_registry_messages_pb2 import (
+    ModelVersion as ProtoModelVersion,
+    ModelVersionStatus as ProtoModelVersionStatus,
+    RegisteredModel as ProtoRegisteredModel,
+    RegisteredModelAlias as ProtoRegisteredModelAlias,
+)
+from mlflow.store._unity_catalog.registry.utils import (
+    model_version_from_uc_proto,
+    registered_model_from_uc_proto,
+)
+
+
+def test_model_version_from_uc_proto():
+    expectedModelVersion = ModelVersion(
```

Comment:
nit: can we use snake case in Python?
```suggestion
    expected_model_version = ModelVersion(
```

Reply:
Same for `actualModelVersion`